src/Security/Authentication/Certificate/src/PACKAGE.md

```diff
@@ -0,0 +1,42 @@
+## About
+
+`Microsoft.AspNetCore.Authentication.Certificate` provides ASP.NET Core middleware that enables an application to support certificate authentication.
+
+## How to Use
+
+To use `Microsoft.AspNetCore.Authentication.Certificate`, follow these steps:
+
+### Installation
+
+```shell
+dotnet add package Microsoft.AspNetCore.Authentication.Certificate
+```
+
+### Configuration
+
+1. Acquire an HTTPS certificate, apply it, and [configure your server](https://learn.microsoft.com/aspnet/core/security/authentication/certauth#configure-your-server-to-require-certificates) to require certificates
```

Comment:
I think the `apply it` part is not clear. What does it mean?
Does it mean install the cert on your machine - trust it basically?

Reply:
Yeah, I'm guessing this refers to installing the cert on the machine. This line was taken directly from [these docs](https://learn.microsoft.com/aspnet/core/security/authentication/certauth#get-started).

Reply:
Thanks Mackinnon. I think it's still worth clarifying it here. But up to you.

Reply:
I've replaced "apply" with "install"